client/my-sites/people/invite-status/index.jsx

```diff
@@ -0,0 +1,51 @@
+import { Button } from '@automattic/components';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+
+const InviteStatus = ( {
+	type,
+	invite,
+	onResend,
+	handleDelete,
+	resendSuccess,
+	requestingResend,
+	inviteWasDeleted,
+	deletingInvite,
+} ) => {
+	const translate = useTranslate();
+	const { isPending } = invite;
+
+	if ( invite && inviteWasDeleted ) {
+		return null;
+	}
+
+	return (
+		<div
+			className={ classNames( 'people-list-item__invite-status', {
+				'is-pending': isPending,
+				'is-invite-details': type === 'invite-details',
+			} ) }
+		>
+			{ isPending && (
+				<Button
+					className={ classNames( 'people-list-item__invite-resend', {
+						'is-success': resendSuccess,
+					} ) }
+					onClick={ onResend }
+					busy={ requestingResend }
```

Comment:
Let's disable this one when `deletingInvite` is true. Now you can resend **_and_** revoke an invite.

Reply:
Same thing as before